frontend/packages/dev-console/integration-tests/features/pipelines/create-from-builder-page.feature

```diff
@@ -7,20 +7,33 @@ Background:
 
 
 @regression, @smoke
-Scenario Outline: Create a basic pipeline from pipeline builder page : P-03-TC08
+Scenario: Pipeline Builder page : P-03-TC02
+    Given user is at pipelines page 
+    When user clicks Create Pipeline button on Pipelines page
+    Then user will be redirected to Pipeline Builder page
+    And Name displayed with default value new-pipeline
```

Comment:
The Name would be in lower case and if it is some label or tag name then it can be in quotes

Reply:
updated as 
user is able to see pipeline name with default value "new-pipeline"